src/main/java/org/gridsuite/explore/server/services/CaseService.java

```diff
@@ -68,6 +68,9 @@ UUID importCase(MultipartFile multipartFile) {
         try {
             caseUuid = restTemplate.postForObject(caseServerBaseUri + "/" + CASE_SERVER_API_VERSION + "/cases/private", request, UUID.class);
         } catch (HttpStatusCodeException e) {
+            if (e.getStatusCode().equals(HttpStatus.UNPROCESSABLE_ENTITY)) {
```

Comment:
create test  with bad extension  to test this block if (e.getStatusCode().equals(HttpStatus.UNPROCESSABLE_ENTITY)) {}

Reply:
ok